src/expression/functions.ts

```diff
@@ -353,6 +353,21 @@ export namespace DefaultFunctions {
         .vectorize(2, [0])
         .build();
 
+    export const trunc = new FunctionBuilder("trunc")
+        .add1("number", n => Math.trunc(n))
+        .add1("null", () => null)
+        .build();
```

Comment:
You could probably also vectorize them by adding `.vectorize(1, [0])`

Reply:
Ah I didn't know what that does so I didn't add it. I'll add it